molecule_common/base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos/systemd:latest
+
+RUN yum makecache fast && yum update -y && \
+    yum install -y python sudo yum-plugin-ovl && \
+    sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf
```

Comment:
Unless we have a reason to keep the yum cache in the image, let's add a `yum clean all` at the end. E.g. https://github.com/sclorg/mongodb-container/blob/master/3.2/Dockerfile#L29